flash/data/batch.py

```diff
@@ -0,0 +1,82 @@
+from typing import Any, Callable, Mapping, Optional, Sequence
+
+import torch
+
+
+class _PreProcessor:
+
+    def __init__(self, collate_fn: Callable, per_sample_transform: Callable, per_batch_transform: Callable):
+        self.collate_fn = collate_fn
+        self.per_sample_transform = per_sample_transform
+        self.per_batch_transform = per_batch_transform
+
+    def __call__(self, samples: Sequence[Any]):
+        samples = [self.per_sample_transform(sample) for sample in samples]
```

Comment:
I think this is an issue it won't support ablumnetations https://github.com/PyTorchLightning/lightning-flash/issues/161

Reply:
No this is not True. This way the user is to implement per_sample_transform as a function by himself. And if he wishes to use albumentations, he has to pass this in accordingly. The default implementation does not do anything here.